demisto_sdk/commands/split/jsonsplitter.py

```diff
@@ -124,4 +146,69 @@ def create_module(self):
 
             module_file_path = os.path.join(self.module_dir, file_name)
 
-        write_dict(module_file_path, data=self.module_json_data, indent=4)
+        write_dict(module_file_path, data=self.json_data, indent=DEFAULT_JSON_INDENT)
+
+    def get_auto_output_path(self) -> Tuple[Path, str, str]:
```

Comment:
does `get_auto_output_path` apply only to lists, or to other content items as well? If it's only for lists, let's mention it in the method name.